app/src/main/java/nodomain/freeyourgadget/gadgetbridge/devices/qhybrid/ConfigActivity.java

```diff
@@ -171,7 +219,8 @@ public void onFinish(boolean success, NotificationConfiguration config) {
                                                 helper.saveNotificationConfiguration(config);
                                                 LocalBroadcastManager.getInstance(ConfigActivity.this).sendBroadcast(new Intent(QHybridSupport.QHYBRID_COMMAND_NOTIFICATION_CONFIG_CHANGED));
                                             } catch (GBException e) {
-                                                e.printStackTrace();
+                                                GB.log("error saving configuration", GB.ERROR, e);
+
                                                 GB.toast("error saving notification", Toast.LENGTH_SHORT, GB.ERROR, e);
```

Comment:
I had tried to say that GB.toast() already logs error(!) messages, so no need to log *and* toast. Sorry for your time adding all those extra log statements :-}
Anyway, we can remove these on demand, they don't really hurt.